Lib/test/test_pydoc/test_pydoc.py

```diff
@@ -2305,6 +2305,33 @@ def test_sys_path_adjustment_when_curdir_already_included(self):
                 trailing_argv0dir = trailing_curdir + [self.argv0dir]
                 self.assertIsNone(self._get_revised_path(trailing_argv0dir))
 
+    def test__get_version(self):
+        import warnings
+        import json
+
+        class Module:
+            __name__ = 'my_module'
+
+            @property
+            def __version__(self):
+                from warnings import _deprecated
+                _deprecated("__version__", remove=(3, 20))
```

Comment:
```suggestion
                warnings._deprecated("__version__", remove=(3, 20))
```

Hopefully it fits on 80 chars.